granadilla/cli.py

```diff
@@ -567,6 +611,45 @@ def extuser_del(self, email):
         account.delete()
 
     @command
+    def device_list(self):
+        """
+        Print the list of devices and their owner.
+        """
+        for device in models.LdapDevice.objects.order_by('device_name'):
+            self.display("%s", device.device_name)
+
+    @command
+    def device_add(self):
+        """
+        Add a new device.
+        """
+        device = models.LdapDevice()
+        user = self.get_device_owner()
+        device.device_owner = user.dn
+        device.device_type = self.grab("Type of device: ", False) 
+        device.device_name = user.username + "_" + device.device_type
```

Comment:
We could instead ask for an explicit name, don't you think? A user might have 2 phones, or 2 laptops while upgrading.